shared/actions/chat2/index.js

```diff
@@ -1893,6 +1894,58 @@ const setConvRetentionPolicy = (action: Chat2Gen.SetConvRetentionPolicyPayload)
   }
   return ret
 }
+
+const setConvExplodingMode = (action: Chat2Gen.SetConvExplodingModePayload) => {
+  const {conversationIDKey, seconds} = action.payload
+  logger.info(`Setting exploding mode for conversation ${conversationIDKey} to ${seconds}`)
+  const cat = Constants.explodingModeGregorKey(conversationIDKey)
+  if (seconds === 0) {
+    // dismiss the category so we don't leave cruft in the push state
+    return Saga.call(RPCTypes.gregorDismissCategoryRpcPromise, {category: cat})
+  }
+  return Saga.call(RPCTypes.gregorInjectItemRpcPromise, {
+    body: seconds.toString(),
```

Comment:
Why toString this?

Reply:
`seconds` is a `number`, and `gregorInjectItem` takes a body that is a `string`

Reply:
Ah I guess it forces you to give a string.